ts/features/wallet/cobadge/component/CobadgeCard.tsx

```diff
@@ -0,0 +1,110 @@
+import * as React from "react";
+import {
+  Image,
+  ImageSourcePropType,
+  ImageStyle,
+  StyleProp,
+  StyleSheet
+} from "react-native";
+import { View } from "native-base";
+import I18n from "../../../../i18n";
+import BaseCardComponent from "../../component/BaseCardComponent";
+import { useImageResize } from "../../onboarding/bancomat/screens/hooks/useImageResize";
+import { H4 } from "../../../../components/core/typography/H4";
+import { H5 } from "../../../../components/core/typography/H5";
+import abiLogoFallback from "../../../../../img/wallet/cards-icons/abiLogoFallback.png";
+
+type Props = {
+  caption?: string;
+  expireMonth?: string;
+  expireYear?: string;
+  abiLogo?: string;
+  brandLogo: ImageSourcePropType;
+};
+
+const styles = StyleSheet.create({
+  abiLogoFallback: {
+    width: 40,
+    height: 40,
+    resizeMode: "contain"
+  },
+  brandLogo: {
+    width: 80,
+    height: 40,
+    resizeMode: "contain"
```

Comment:
```suggestion
    width: 48,
    height: 30,
    resizeMode: "contain"
```

Changed following the Invision specs:
![Schermata 2021-01-29 alle 18 18 35](https://user-images.githubusercontent.com/26501317/106306471-6839d300-625e-11eb-8446-9704ff0dc244.png)

Reply:
Modified dimensions accordingly to the InVision specs in [8e95104](https://github.com/pagopa/io-app/pull/2751/commits/8e9510497ff5b0c333b56f55bfbeb568b8a6c38c)